recipe/meta.yaml

```diff
@@ -42,9 +45,17 @@ test:
     - pytest
     - pytest-xdist
     - mpmath
+    - curl      # [win]
+    - m2-unzip  # [win]
 {% if ver_major_minor == "1.5" %}
     - blas * netlib  # [ppc64le]
 {% endif %}
+  commands:
+    - curl -L -O https://github.com/lucasg/Dependencies/releases/download/v1.10/Dependencies_x64_Release.zip  # [win]
+    - unzip Dependencies_x64_Release.zip  # [win]
+    - Dependencies.exe -imports %SP_DIR%\scipy\linalg\_fblas.cp39-win_amd64.pyd  # [win]
+    - set "PATH=%SP_DIR%\scipy\.libs;%PATH%"  # [win]
```

Comment:
I don't know why this is needed in python 3.9 windows. I even tried adding `os.add_dll_directory` in `_distributor_init.py`.
cc @rgommers, @mingwandroid

Reply:
Was just looking at this PR again, and I believe it's because of this conda bug: https://github.com/conda/conda/issues/10897